diff --git a/node_modules/react-native-intent-launcher/android/build.gradle b/node_modules/react-native-intent-launcher/android/build.gradle
index ae3c73b..8e8929f 100644
--- a/node_modules/react-native-intent-launcher/android/build.gradle
+++ b/node_modules/react-native-intent-launcher/android/build.gradle
@@ -16,7 +16,7 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: "libs", include: ["*.jar"])
-    compile "com.android.support:appcompat-v7:+"
-    compile "com.facebook.react:react-native:+"  // From node_modules
+    implementation fileTree(dir: "libs", include: ["*.jar"])
+    implementation "com.android.support:appcompat-v7:+"
+    implementation "com.facebook.react:react-native:+"  // From node_modules
 }
